xapian-core/cluster/cluster.cc

```diff
@@ -54,41 +60,46 @@ MSetDocumentSource::MSetDocumentSource(const MSet & mset_, doccount maxitems_)
 
 Document
 MSetDocumentSource::next_document() {
```

Comment:
Xapian convention is that the opening `{` should go on a new line here.  Generally we "cuddle" braces in the interests of conserving vertical space so more code fits on the screen.  The exception here is mostly just because it ends up cleaner with long parameter lists to start a new line.  And the "fit more on the screen" argument is more important within a function (or a class, struct, etc).